usr/lib/NetworkManager/conf.d/mac-address.conf

```diff
@@ -0,0 +1,2 @@
+[device]
```

Comment:
Should this file have a priority? Perhaps higher than what is already in the folder. I see the following on a clean install of PlaytronOS:
 - 20-connectivity-fedora.conf
 - 22-wifi-mac-addr.conf
 
The contents of `22-wifi-mac-addr.conf` seems to indicate it is responsible for the generated MAC addresses.
 
Maybe this file could be something like `50-playtron.conf`

Reply:
Good catch! Let me change that.